Mobile_Detect.php

```diff
@@ -1145,8 +1145,8 @@ protected function matchUAAgainstKey($key)
     /**
      * Check if the device is mobile.
      * Returns true if any type of mobile device detected, including special ones
-     * @param  null $userAgent   deprecated
-     * @param  null $httpHeaders deprecated
+     * @param  string $userAgent   deprecated
```

Comment:
The params are deprecated so we want them to be considered as `null` by IDEs

Reply:
IHMO they may be deprecated, but still work on array and string types, just like in isTablet method below.
To mark it for IDE, people used the following approach https://stackoverflow.com/a/27005644

Reply:
OK, I'll use setUserAgent, thank you.